### DISCUSSION TITLE
How to Display Full Directory Path in oh-my-bash Prompt Without Truncation?

### DISCUSSION THREAD

**Discussion by klsbecker** [ohmybash/oh-my-bash#540] [General]
Hello everyone,

I've been using oh-my-bash for a while, and I've come across a minor annoyance that I'm hoping to find a solution for. By default, when navigating deeply nested directories, oh-my-bash truncates the path in the prompt with ellipses (...), like so: `kls@APED14:~/.../rts/3S$`. This behavior makes it challenging to quickly understand my current directory context at a glance.

I'm wondering if anyone knows a way to configure oh-my-bash to display the full directory path in the prompt without truncating it, regardless of its length. I've looked through the configuration options but haven't found a clear way to achieve this. Any tips or guidance would be greatly appreciated. I'm sure this adjustment would be beneficial for those of us who prefer or require full path visibility for our workflow.

Thanks in advance for any assistance or advice you can provide!

**Comment by akinomyoga** [comment]
you can set `PROMPT_DIRTRIM=0` after `source "$OSH"/oh-my-bash.sh` in your `~/.bashrc`.

**Reply by klsbecker** [reply]
It works! Thanks for the help.
